wp-content/plugins/goonj-blocks/build/render.php

```diff
@@ -24,13 +24,28 @@
 
 $heading_text = $headings[$target];
 
-$register_link = sprintf(
+$collection_camp_register_link = sprintf(
```

Comment:
The register link here is currently used for volunteer registration, and right now it only works for collection camps. However, we also need this functionality for institution collection camps and dropping center. To achieve this, we're passing different state and city IDs as parameters, which is why we need to update the code. If you check the code below, you'll see how it can be dynamically adjusted based on these parameters. Let me know if you have any questions; we can also get on a call, and I can give you a demo to explain everything.